### ISSUE TITLE
About l_pix during training the sr model

### ISSUE THREAD

**Issue by ztt0821** [xinntao/BasicSR#29]
When I use the your model to train the sr model. I found a problem, the l_pix is stable at around 0.02, is this normal? I have not changed any parameters, it is the same as your model.

**Comment by ztt0821** [comment]
Thank you very much. The l_pix is the pixel loss?

**Comment by xinntao** [comment]
Yes.

**Comment by ackbar03** [comment]
@xinntao  hi xinntao, I see in your loss_log file the reduction in loss is very small, it starts out around the range 0.03-0.04 and ends around 0.01-0.02 range. 

Is there a large benefit in continuing to train for over 800k iterations? Do you see any notable improvements/changes when using the model for application due to the ~0.01 reduction in loss? Also is this typical of most deep learning training processes? 

I'm asking because my experience is still relatively limited. From the experiments I've run before, sometimes the change in loss is very noticeable after long iterations but sometimes it is also very minimal like this case and I'm not sure whether it should continue running or whether there is little difference stopping it say at around 200-400k iterations.

**Comment by xinntao** [comment]
Hi @ackbar03 
The loss indeed becomes very small after some training, especially in the later stage.
I attached a validation curve:
![image](https://user-images.githubusercontent.com/17445847/50376129-df461b80-0643-11e9-90e7-442b02c34191.png)
For the curve, we can see that there is still improvement after 800k, from (800k, 30.24dB) to (1M, 30.27dB), though the improvement 0.03dB is small and you must see no changes visually.

Though there is small or even no obvious drop in training loss, it seems it can still perform better in validation with more training. 
And I have not try many learning schemes. I think there must be some better learning schemes with less training iterations.
